provider/cmd/pulumi-resource-equinix/schema.json

```diff
@@ -12810,7 +12810,7 @@
    Package = new Equinix.Fabric.Inputs.CloudRouterPackageArgs\n        {\n            Code = \"STANDARD\",\n        },\n        Project = new Equinix.Fabric.Inputs.CloudRouterProjectArgs\n        {\n            ProjectId = \"776847000642406\",\n        },\n        Account = new Equinix.Fabric.Inputs.CloudRouterAccountArgs\n        {\n            AccountNumber = 203612,\n        },\n    });\n\n});\n```\n```java\npackage generated_program;\n\nimport com.pulumi.Context;\nimport com.pulumi.Pulumi;\nimport com.pulumi.core.Output;\nimport com.pulumi.equinix.fabric.CloudRouter;\nimport com.pulumi.equinix.fabric.CloudRouterArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterNotificationArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterOrderArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterLocationArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterPackageArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterProjectArgs;\nimport com.pulumi.equinix.fabric.inputs.CloudRouterAccountArgs;\nimport java.util.List;\nimport java.util.ArrayList;\nimport java.util.Map;\nimport java.io.File;\nimport java.nio.file.Files;\nimport java.nio.file.Paths;\n\npublic class App {\n    public static void main(String[] args) {\n        Pulumi.run(App::stack);\n    }\n\n    public static void stack(Context ctx) {\n        var newCloudRouter = new CloudRouter(\"newCloudRouter\", CloudRouterArgs.builder()\n            .name(\"Router-SV\")\n            .type(\"XF_ROUTER\")\n            .notifications(CloudRouterNotificationArgs.builder()\n                .type(\"ALL\")\n                .emails(                \n                    \"example@equinix.com\",\n                    \"test1@equinix.com\")\n                .build())\n            .order(CloudRouterOrderArgs.builder()\n                .purchaseOrderNumber(\"1-323292\")\n                .build())\n            .location(CloudRouterLocationArgs.builder()\n                .metroCode(\"SV\")\n                .build())\n            .package_(CloudRouterPackageArgs.builder()\n                .code(\"STANDARD\")\n                .build())\n            .project(CloudRouterProjectArgs.builder()\n                .projectId(\"776847000642406\")\n                .build())\n            .account(CloudRouterAccountArgs.builder()\n                .accountNumber(\"203612\")\n                .build())\n            .build());\n\n    }\n}\n```\n```yaml\nresources:\n  newCloudRouter:\n    type: equinix:fabric:CloudRouter\n    name: new_cloud_router\n    properties:\n      name: Router-SV\n      type: XF_ROUTER\n      notifications:\n        - type: ALL\n          emails:\n            - example@equinix.com\n            - test1@equinix.com\n      order:\n        purchaseOrderNumber: 1-323292\n      location:\n        metroCode: SV\n      package:\n        code: STANDARD\n      project:\n        projectId: '776847000642406'\n      account:\n        accountNumber: '203612'\n```\n{{% /example %}}\n\n{{% /examples %}}",
```

Comment:
In this case it appears the change is that the old description had examples that referenced `CloudRouterNotificationArgs`, but the new description has some sort of generic object definition `{ "key": "value" ... }` instead.  Why did `make tfgen` produce references to CloudRouterNotificationArgs but `make only_tfgen` doesn't?

Reply:
Currently the `tfgen` task is defined as:

```make
tfgen: only_tfgen generate_examples
```

If `only_tfgen` updates the schema, and that schema includes examples that are changed by `generate_examples`, perhaps we need to redefine the `tfgen` task so that it runs `only_tfgen` to update the schema JSON so that `generate_examples` generates the correct examples and then runs `only_tfgen` after the examples are generated to make sure any changes to the examples are reflected in the schema JSON.

Reply:
I tried adding just`build_schema` after `examples` and it works well without dirty versions

`generate_examples: examples build_schema`